app/Providers/AppServiceProvider.php

```diff
@@ -24,6 +26,9 @@ public function boot()
     {
         Schema::defaultStringLength(191);
 
+        $driver = DB::getDriverName();
+        $this->loadMigrationsFrom(database_path("migrations/$driver"));
+
```

Comment:
As Laravel supports PostgreSQL, out-of-the-box this shouldn't be necessary, as Eloquent and the query builder know how to form it.